src/Deprecations.jl

```diff
@@ -280,3 +280,8 @@ end
 @deprecate prime_field(L::Union{QadicField, LocalField}) absolute_base_field(L)
 @deprecate coefficient_ring(k::LocalField) base_field(k)
 @deprecate coefficient_field(k::QadicField) base_field(k)
+
+# Deprecated in 0.35.10
+@deprecate minpoly(a::Union{LocalFieldElem, QadicFieldElem, RelFinFieldElem, AbsNumFieldOrderElem}, R::PolyRing) minpoly(R, a)
+@deprecate minpoly(a::AbsNumFieldOrderElem, R::PolyRing) minpoly(R, a)
+@deprecate charpoly(a::RingElem, R::PolyRing) charpoly(R, a)
```

Comment:
for charpoly holds the same, i.e. better to use a union here. and for minpoly, the single one in line 286 is also contained in the union the line above

Reply:
Oops, that was a typo. For `minpoly` I only want `AbsSNumFieldOrderElem`.